### ISSUE TITLE
Fatal Error: Could not find lock file: Failed to parse lockfile toml. 

### ISSUE THREAD

**Issue by codehz** [wasmerio/wapm-cli#76]
Thanks for the bug report!

### Describe the bug

race condition in wapm run

Copy and paste the result of executing the following in your shell, so we can know the version of `wapm-cli`, Rust (if available) and architecture of your environment.

```bash
wapm-cli 0.1.0
rustc 1.35.0-beta.1 (2bc1d406d 2019-04-10)
```

### Steps to reproduce

```zsh
#!/bin/zsh
wapm install cowsay lolcat
while wapm run cowsay "Hello" | wapm run lolcat; do :;done
```

### Expected behavior

```
 _______
< Hello >
 -------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
               ||----w |
                ||     ||
```
(endless loop)

### Actual behavior

```
Fatal Error: Could not find lock file: Failed to parse lockfile toml. Did you modify the generated lockfile? Toml error: Error { inner: ErrorInner { kind: Custom, line: None,
 col: 0, message: "missing field `modules`", key: [] } }
```
(about 5%)

**Comment by xmclark** [comment]
Thank you for the bug report! I'm glad someone was able to stress test the tool like this.

**Comment by syrusakbary** [comment]
This is fixed in the latest version of wapm (0.2.0).
Closing the issue :)
